src/engine/framework/System.cpp

```diff
@@ -282,6 +282,7 @@ static void Shutdown(bool error, Str::StringRef message)
     Application::Shutdown(error, message);
 
 	// Always run CON_Shutdown, because it restores the terminal to a usable state.
+	Cvar_ClearCvarMap();
```

Comment:
This should be above the comment, in an if(!error) block. When an error occurs we want to just quit as fast as possible (but need to reset the console to avoid garbling it).